File: java/seasar/annotation/src/test/Tester.java
package test;

import org.seasar.framework.container.*;
import org.seasar.framework.container.factory.*;

public class Tester {

	private final static String PATH = "test.dicon";

	public static void main(String[] args) {

		S2Container container = S2ContainerFactory.create(PATH);
		container.init();

		try {

			printOut((Product)container.getComponent("product1"));

		} finally {
			container.destroy();
		}
	}

	private static void printOut(Product product) {
		System.out.println("----- Product -----");
		product.printOut();
		System.out.println("");
	}
}

File: csharp/AspectDNG+S2.NET/extend-s2container/Tester.cs
using System;

using Seasar.Framework.Container;
using Seasar.Framework.Container.Factory;

public class Tester
{
	private const string PATH = "app.dicon";
	
	public static void Main(string[] args)
	{
		IS2Container s2c = S2ContainerFactory.Create(PATH);

		DataProcessor processor = s2c.GetComponent("processor") as DataProcessor;
		processor.Process();
	}
}